docs/build-the-docs.md

```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
     -v $(pwd):$(pwd) \
     -w $(pwd) \
     antora/antora:1.0.1 \
-    generate site.yml
+    generate \
+        --clean \
+        --pull \
+        --quiet \
+        --silent \
+        --ui-bundle-url <url-or-path-to-ui-bundle>i \
```

Comment:
``--ui-bundle-url <url-or-path-to-ui-bundle>i \`` --> ``--ui-bundle-url <url-or-path-to-ui-bundle.zip> \``

Reply:
Oh naff, I didn't see that I'd made that mistake. Thanks.